src/ui/control/geolocate_control.js

```diff
@@ -122,8 +136,11 @@ class GeolocateControl extends Evented {
             '_finish',
             '_setupUI',
             '_updateCamera',
-            '_updateMarker'
+            '_updateMarker',
+            '_updateMarkerRotation'
         ], this);
+
+        this._updateMarkerRotationThrottled = throttle(this._updateMarkerRotation, 20);
```

Comment:
Is the `deviceorientation` event called much more often than 20ms to necessitate throttling? Just curious.

Reply:
@mourner The motion sensor frequency can be uncapped due to this bug https://w3c.github.io/deviceorientation/spec-source-orientation.html#security-and-privacy

Some say it's suggested to 60 Hz (equalling once every 16.6ms) but I couldn't find the source of this suggestion.